src/theme/NavbarItem.tsx

```diff
@@ -10,45 +10,54 @@ enum ProtocolVersion {
   V3 = 'V3',
 }
 
+const UNKNOWN_PROTOCOL_VERSION = 'UNKNOWN_PROTOCOL_VERSION'
+const FALLBACK_DOCS_VERSION = ProtocolVersion.V3
+
 const getSection = (version: string) => {
-  if (version === ProtocolVersion.V2) {
-    return DocsProtocolVersion.V2
+  switch (version) {
+    case ProtocolVersion.V1:
+      return DocsProtocolVersion.V1
+    case ProtocolVersion.V2:
+      return DocsProtocolVersion.V2
+    case ProtocolVersion.V3:
+      return DocsProtocolVersion.V3
+    default:
+      return UNKNOWN_PROTOCOL_VERSION
   }
+}
 
-  if (version === ProtocolVersion.V1) {
-    return DocsProtocolVersion.V1
+const getSelectedDocVersion = (docVersion: string) => {
+  switch (docVersion) {
+    case ProtocolVersion.V1:
+    case '1.0.0':
+      return ProtocolVersion.V1
+    case ProtocolVersion.V2:
+    case '2.0.0':
+      return ProtocolVersion.V2
+    default:
+      return FALLBACK_DOCS_VERSION
```

Comment:
Can we add an explicit V3? If we change the default we could lose the V3 capture